src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/_help.py

```diff
@@ -26,17 +26,17 @@
     short-summary: Manage Azure EventHubs Authorizationrule connection strings for Namespace
 """
 
-helps['eventhubs namespace network-ruleset'] = """
+helps['eventhubs namespace network-rule-set'] = """
```

Comment:
This should be `network-rule` not `network-rule-set` or `network-ruleset`

Reply:
@tjprescott ,  My bad.  I have pushed the changes as per the guideline. 

I am not sure how to below errors, as these are custom methods:
Command: `eventhubs namespace network-rule ip-address-rule list` - List commands should not expose --ids argument
    Command: `eventhubs namespace network-rule list` - List commands should not expose --ids argument
    Command: `eventhubs namespace network-rule virtual-network-rule list` - List commands should not expose --ids argument